pkg/controller/configmap/configmap_controller.go

```diff
@@ -196,6 +196,13 @@ func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Res
 			}
 		}
 	}
+    networkConfigChanged := true
+    if r.sharedInfo.NetworkConfig != nil {
+        networkConfigChanged = HasNetworkConfigChange(networkConfig, r.sharedInfo.NetworkConfig)
```

Comment:
nit: `HasNetworkConfigChange` -> `hasNetworkConfigChanged`?

Reply:
You're right but I kept it capital for consistency with NeedAppluChange

Reply:
initially I assumed HasNetworkConfigChange would return an error, but now it's clear I can simply remove this code.